airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/orchestration/db/direct_load_table/AlterTableReport.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.orchestration.db.direct_load_table
```

Comment:
moved the existing AlterTableReport from typing_deduping -> here. afaict bigquery is the only class that even used the old CDK's AlterTableReport; maybe this will be useful for other connectors.